WARNING: python/3.6.5 cannot be loaded due to a conflict.
HINT: Might try "module unload anaconda" first.
Traceback (most recent call last):
  File "/public1/home/sc30830/software/alamode/alamode-1.1.0/tools/displace.py", line 21, in <module>
    import numpy as np
ModuleNotFoundError: No module named 'numpy'
Traceback (most recent call last):
  File "/public1/home/sc30830/software/alamode/alamode-1.1.0/tools/extract.py", line 22, in <module>
    import numpy as np
ModuleNotFoundError: No module named 'numpy'
/var/spool/slurm/d/job1493498/slurm_script: line 174: 24926 Segmentation fault      alm alm.in > alm2.log
mv: cannot stat ‘uc.fcs’: No such file or directory
mv: cannot stat ‘uc.xml’: No such file or directory
